packages/store/src/store.ts

```diff
@@ -41,19 +41,17 @@ export class Store {
     const selectorFn = getSelectorFn(selector);
     return this._stateStream.pipe(
       map(selectorFn),
-      catchError(
-        (err: Error): Observable<never> | Observable<undefined> => {
-          // if error is TypeError we swallow it to prevent usual errors with property access
-          const { suppressErrors } = this._config.selectorOptions;
+      catchError((err: Error): Observable<never> | Observable<undefined> => {
```

Comment:
Try to keep changes unrelated to the PR out of the commits if possible, it just makes reviewing more difficult to see these changes and to try to figure out if anything has actually changed here.
These format changes, while valid, still cause a bit of confusion as to how this relates to the PR.

Reply:
done